testgrid/config/README.md

```diff
@@ -183,6 +183,19 @@ test_groups:
   num_columns_recent: 3
 ```
 
+### See 'Summary' first
+If you'd like to see the 'Summary' tab first when opening a dashboard, instead
+of the first tab of results:
```

Comment:
This is actually going to be completely default with no option, so going to remove this section from the README